assets/js/client.es6.js

```
@@ -421,6 +428,30 @@ function initialize(bindLinks) {
     cookies.set('over18', val)
   });
 
+  app.on(constants.HIDE_GLOBAL_MESSAGE, function(message) {
+    let options = {
+      expires: new Date(message.expires),
+    };
+    cookies.set(message.key, 'globalMessageSeen', options);
```

May want to throw an error if message.key is undefined